### PR TITLE
DAOS-11189 test: support multiple dfuse instances

### PR DESCRIPTION
**Pull request by daltonbohning**
Test-tag: dfuse,pr dfuse,daily_regression test_dfuse_find
Skip-unit-tests: true
Skip-fault-injection-test: true

- Add standalone methods to start dfuse
- Adjust dfuse test base to use new methods
- Adjust dfuse command sort order to be user friendly
- Fix dfuse/find.py to use new methods

Signed-off-by: Dalton Bohning <dalton.bohning@intel.com>

Required-githooks: true
Signed-off-by: Dalton Bohning <dalton.bohning@intel.com>

### CODE REVIEW

**Comment by daltonbohning** [comment]
Existing test failures. https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-9848/5/tests
- `dfuse/fio_small.py` - https://daosio.atlassian.net/browse/DAOS-11377

File: src/tests/ftest/util/dfuse_utils.py [hunk 1]
```diff
@@ -37,6 +36,34 @@ def __init__(self, namespace, command):
...
+
+        Returns:
+            int: the sort priority
+
+        """
+        return {
+            'mount_dir': 1,
+            'puuid': 2,
+            'cuuid': 3
+        }.get(key, 0)
```
Comment (daltonbohning) [inline src/tests/ftest/util/dfuse_utils.py:54]: This is so the args are ordered like this for readability readability:
```
dfuse --mounpoint <mountpoint> --pool <pool> --container <cont>
```

**Review by shimizukko — DISMISSED** [review]

**Comment by daltonbohning** [comment]
Ping reviewers

**Comment by daltonbohning** [comment]
Test passed in latest run - ready for review again

**Review by shimizukko — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]

File: src/tests/ftest/util/dfuse_utils.py [hunk 2]
```diff
@@ -410,3 +410,47 @@ def stop(self):
 
         else:
             self.log.info("No hosts running dfuse - nothing to stop")
+
+
+def get_dfuse(test, hosts):
```
Comment (ashleypittman) [inline src/tests/ftest/util/dfuse_utils.py:415]: Didn't I make a change similar to this function myself recently?

**Review by ashleypittman — APPROVED** [review]

File: src/tests/ftest/util/dfuse_utils.py (hunk 2 above)
Comment (daltonbohning) [reply src/tests/ftest/util/dfuse_utils.py:415]: Yes. You added `DfuseTestBase.load_dfuse`, which initializes `self.dfuse`. This new function initializes a dfuse instance independently of `self.dfuse` so tests can instantiate multiple instances

**Pull request merged**